app/controllers/events_controller.rb

```diff
@@ -0,0 +1,55 @@
+class EventsController < ApplicationController
+  after_action :invite_creator, only: [:create]
+  def index
+    @events = Event.all.order('created_at DESC')
+    @upcoming_events = Event.upcoming_events
+    @prev_events = Event.prev_events
+
+    redirect_to new_session_path unless signed_in?
+  end
+
+  def new
+    @event = User.find(current_user_id).events.build
```

Comment:
- [ ] Why not just `current_user.events.build`? You already have this method, and I think it's more readable.
-- Same in create action